kettle-plugins/formats/src/main/resources/org/pentaho/big/data/kettle/plugins/formats/parquet/messages/messages_en_US.properties

```diff
@@ -0,0 +1,9 @@
+BaseStepDialog.StepName=Step name:
```

Comment:
I'm on linux. Do you remember where that was? We have a lot of unescaped colons in kettle